draft-ietf-mops-streaming-opcons.md

```diff
@@ -341,6 +371,60 @@ We note that other operators saw similar spikes during this time period. Craig L
 - A 30% increase in upstream traffic over their pre-pandemic levels, and
 - A steady increase in the overall volume of DDoS traffic, with amounts exceeding the pre-pandemic levels by 40%. (He attributed this increase to the significant rise in gaming-related DDoS attacks ({{LabovitzDDoS}}), as gaming usage also increased.)
 
+# Latency Considerations
+
+Streaming media latency refers to the "glass-to-glass" time duration, which is the delay between the real-life occurence of a recorded event and the media being appropriately displayed on an end user's device.  Note that this is different from the network latency (defined as the time for a packet to cross a network from end to end) because it includes video encoding time, and for most cases also ingest to an intermediate service such as a CDN or other video distribution service, rather than a direct connection to an end user.
+
+Streaming media can be usefully categorized according to the application's latency requirements into a few rough categories:
+
+  * ultra low latency    (less than 1 second)
+  * low latency live     (less than 10 seconds)
+  * non-low-latency live (10 seconds to a few minutes)
+  * on-demand            (hours or more)
```

Comment:
This might be better described as precise latency not being part of on-demand service level objectives.

I also get the impression that the expectation of customers is that live streams are consumed more or less simultaneously by users, while on-demand streams carry the expectation that viewers could be anywhere in the stream at any moment. This implies something about the acceptable latency, but I'm not sure the latency is the defining characteristic. Happy to discuss more at the interim.

Reply:
I recall in a talk at MileHighVideo in August 2019 on LL streaming that to get there LL services are delivering simultaneously 2 levels of latency to users.   One class gets the fastest LL.  Users get added into this fast LL class until that serving resource gets to its load level.  A second class of higher latency (slower) users are then created using a different delivery that isn't as fast, but scales to more users.      

This aspect may be worth documenting as handling migration/handoff between these two classes could be a use case ultimately.

Reply:
Addressing the on-demand definition in the new PR.